Realm.podspec

```diff
@@ -52,7 +52,8 @@ Pod::Spec.new do |s|
   s.osx.vendored_library    = 'core/librealm.a'
 
   s.subspec 'Headers' do |s|
-    s.source_files          = 'include/**/*.h'
+    s.source_files          = 'include/**/*.{h,hpp}'
```

Comment:
That's more or less a limitation of CocoaPods. It's the only possible way to configure the `*.hpp` headers in the podspec to have the visibility **project**. It's not possible to set that directly. (Intentionally, to avoid conflicting configurations) Otherwise we would need to duplicate the public and private header file definitions on the root podspec, this will make the podspec more complex and cause that the headers are added twice in the project / file ref tree, which is unfortunate for users inspecting the target. /c @segiddins